easy-pbkdf2.js

```diff
@@ -105,7 +105,7 @@ EasyPbkdf2.prototype = {
 		var defaultHashIterations = this.DEFAULT_HASH_ITERATIONS,
 			saltSize = this.SALT_SIZE;
 
-		if ( !callback && _.isFunction( explicitIterations ) ) {
+		if ( !callback && explicitIterations instanceof Function ) {
```

Comment:
`instanceof` checks are usually avoided because they will fail for values from other realms (vm, iframes, windows).